controllers/cartController.js

```diff
@@ -0,0 +1,20 @@
+const { catchAsync } = require("../utils/error");
+const cartService = require("../services/cartsService");
+
+const getCartList = catchAsync(async (req, res) => {
+  const list = await cartService.getCartList(req.userId);
+
+  res.status(201).json({ data: list });
+});
+
+const deleteCart = catchAsync(async (req, res) => {
+  const { cartId } = req.query;
```

Comment:
```suggestion
  const { cartIds } = req.query;
```
cart_id가 하나만 들어오면 하나만 삭제하고, 여러개 선택되거나, 전체 선택되었을때도 모두 처리할 수 있도록 해야합니다. 이를 위해서 변수명 자체를 우선 cartIds라는 이름으로 복수 데이터도 의미할 수 있도록 바꿔주세요.